startup_runners/Makefile

```diff
@@ -19,14 +19,14 @@ startup_runner.class: startup_runner.java
 
 
 outer_startup_runner_c: outer_startup_runner.c
-	${CC} ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
-		${CPPFLAGS} outer_startup_runner.c \
+	${CC} --std=c99 ${C_ITER_RUNNER_CFLAGS} ${CFLAGS} -L${PWD}/../krun/libkrun \
```

Comment:
I guess that's OK, as long as calling c99 code from gnu99 code is OK. I don't see why not.